spec/features/posting_peeps_spec.rb

```diff
@@ -0,0 +1,8 @@
+feature 'posting peeps' do
```

Comment:
I like your tests, they're super clear and easy to read. Also very detailed making it easy to pick up your code really fast.